### ISSUE TITLE
Downloadrequest finished before Calculate

### ISSUE THREAD

**Issue by NilsPvR** [NilsPvR/Gleitzeitkonto-Browser#15]
Currently both requests to the Gleitzeitkonto-API (calculate and download) are sent at the same time. Based on once the promise for the requests resolves the display will be updated.
Since normally the download takes way longer the download will always update the display after the calculate request has updated the display. See these two lines in [Gleitzeitkonto-Browser.js](https://github.com/NilsPvR/Gleitzeitkonto-Browser/blob/main/Extension/Gleitzeitkonto-Browser.js)
```js
time.updateDisplay(promiseCalcKontoData, true);
time.updateDisplay(promiseDownloadKontoData, false);
```

The issue occurs when the webserver is not started. Then it's a race to which request finished earlier. If the calculate finishes afterwards then the display will forever show loading and the button to reload will be disabled.

![grafik](https://user-images.githubusercontent.com/58894308/225236987-ba15ca81-f384-45f0-bdaa-0b8f70f23568.png)

**Comment by NilsPvR** [comment]
Is actually fixed with 88652d3 since now only the latest data will be shown.

**Comment by NilsPvR** [comment]
Has been merged to main with a763452
